Aptoide/appcoins-wallet-android
# [APPC-1189] Click on url without browser

**What does this PR do?**

   This PR fixes 2 bugs. When the user clicks on an url that would open a browser, the app would crash if the user didn’t have any browser
  The other fix is related to changes that were done to fix a crash that would happen if you would press the buy button too fast. The solution was to remove some setText and image on the expressCheckoutFragment, but since there was some hardcoded values on the view, those values would show app. This PR, makes it so that the view appears empty.
This PR also adds mergePath for a lottie animation that is crashing on some devices (doesn’t fix the problem but it’s better to have it)

**Where should the reviewer start?**

ExternalBrowserRouter.java

**How should this be manually tested?**

  Disable browser, go to settings, press the telegram button, it should show a toast saying unknown error
 Buy oil, the inbetween screen should not have nothing on it
 Do a credit card transaction on buy gas, and check if the sucess animation breaks (except on the motorolla, that one it’s not fixed, the others is just fail safe as it should not break)

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1189

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
